app/forms/hyrax/performance_form.rb

```diff
@@ -4,26 +4,24 @@ module Hyrax
   # Generated form for Performance
   class PerformanceForm < Hyrax::Forms::WorkForm
     self.model_class = ::Performance
-    self.terms -= [:creator, :keyword, :subject, :license, :rendering_ids]
-    self.terms += [:composition_id, :instruments, :date, :tags,
-                   :length, :genre, :software, :medium, :performer]
+    self.terms += [:composition_id, :venue, :date, :duration,
+                   :resource_type, :medium]
 
-    self.required_fields = [:title, :instruments]
+    self.required_fields = [:title, :subject]
```

Comment:
Consider adding `:creator` to the `required_fields`.

Reply:
Will update it